docs/docs/dsl.md

```diff
@@ -375,6 +375,16 @@ HttpRoutes.of[IO] {
 }
 ```
 
+**Please note:** You cannot mix left- and right-associative matchers in a path! So something like `case GET -> "hello" / "world" /: rest => ???` will **not compile**.
```

Comment:
Actually, I'm not that much UI designer, but if something doesn't compile, I'd argue we shouldn't highlight this much 😄 Also, perhaps, we can mark that this behavior is _expected_ rather than point out _failing compilation_. WDYT?

Reply:
I removed the bold one on `not compile`
For the rest I'd say we stick with `cannot` because at least from my perspective I expected to be able to mix both or have it working by throwing around parentheses. ;-)